test/connect-proxy/device.ts

```diff
@@ -26,7 +26,9 @@ const { expect } = chai;
 nock.disableNetConnect();
 
 const BALENA_API_HOST = process.env.BALENA_API_HOST!;
-const VPN_SERVICE_API_KEY = Buffer.from(process.env.VPN_SERVICE_API_KEY!);
+const VPN_SERVICE_API_KEY = Buffer.from(
+	process.env.VPN_SERVICE_API_KEY || 'test_vpn_string',
```

Comment:
I think it's better to fail if the env var is not set than falling back to an arbitrary default that only really makes sense in tests. The way it is now, it may mask misconfiguration.

Reply:
I'm not sure I follow, this is only for tests which will be run under different config environments anyway. Adding the default here was to reduce the required `ENV` lines in the `Dockerfile` when running the tests when building (which itself is a workaround so the CI runs the tests).

Reply:
Oh, yeah, I didn't realise that file was a test file :)